### ISSUE TITLE
It wont open

### ISSUE THREAD

**Issue by Foxiseminger** [Alphatism/loquibot#9]
i opened the setup and wont open

**Comment by Foxiseminger** [comment]
help @Alphalaneous

**Comment by Alphalaneous** [comment]
if the setup itself isn’t opening:
Smart Screen popup: press read more, then run anyways
Doesn’t open at all, no popup, check your antivirus

**Comment by FoxyWorld1** [comment]
No, it straight up does not open setup. I tried portable but no luck.

**Comment by FoxyWorld1** [comment]
also I'm fox's alt account
